src/ekore/operator_matrix_elements/polarized/space_like/as2.py

```diff
@@ -0,0 +1,472 @@
+r"""
+This module contains the |NNLO| |OME| in the polarized case for the matching conditions in the |VFNS|.
+The equations given in :cite:`Bierenbaum_2023`. In the paper the fraction :math:`\mu_F^2 / m_H^2` inside the log is inverted, yielding an additional factor of -1 wherever L has an odd power. The anomalous dimensions are expressed with an additional factor -1 as well relative to the paper. The anomalous dimensions with the addition 'hat', have the form :math:`\gamma_hat = gamma(nf+1) - gamma(nf)`.
+"""
+import numba as nb
+import numpy as np
+
+from eko import constants
+
+from ....harmonics.constants import zeta2, zeta3
+from ....harmonics.polygamma import cern_polygamma
+
+
+@nb.njit(cache=True)
+def A_qq_ns(n, sx, L):
+    r"""
+    |NNLO| light-light non-singlet |OME| :math:`A_{qq,H}^{NS,(2)}` given in
+    Eq. (133) in :cite:`Bierenbaum_2023`
+
+    Parameters
+    ----------
+        n : complex
+            Mellin moment
+        sx : list
+            harmonic sums cache
+        L : float
+            :math:`\ln(\mu_F^2 / m_h^2)`
+
+    Returns
+    -------
+        A_qq_ns : complex
+            |NNLO| light-light non-singlet |OME| :math:`A_{qq,H}^{NS,(2)}`
+    """
+    S1 = sx[0][0]
+    S2 = sx[1][0]
+    S3 = sx[2][0]
+
+    gamma0_qq = (
+        -2 * constants.CF * (3 - 4 * S1 + 2 / (n * (n + 1)))
+    )  # relative to the paper theres a factor -1 (anomalous = -splitting)
+    gamma1_qqNS = (
+        -constants.CF
+        * constants.TR
+        * (4 / 3)
+        * (
+            -8 * S2
+            + (40 / 3) * S1
+            - (3 * n**4 + 6 * n**3 + 47 * n**2 + 20 * n - 12)
+            / (3 * n**2 * (n + 1) ** 2)
+        )
+    )  # relative to the paper theres a factor -1
```

Comment:
```suggestion
    ) # relative to the paper there is a factor -1
```
In case you wanted to delete these comments - keep them, they don't hurt and make things more clear ...